native/build.sbt

```diff
@@ -13,6 +15,12 @@ scalaJSModuleKind in Test := ModuleKind.CommonJSModule
 
 jsEnv in Test := new NodeJSEnv() {
   override def customInitFiles(): Seq[VirtualJSFile] = super.customInitFiles() :+ new MemVirtualJSFile("addReactNativeMock.js").withContent(
-    s"""require("${(baseDirectory.value / "node_modules/react-native-mock-render/mock.js").getAbsolutePath}");"""
+    s"""require("${escapeBackslashes((baseDirectory.value / "node_modules/react-native-mock-render/mock.js").getAbsolutePath)}");"""
   )
 }
+
+def escapeBackslashes(path: String): String =
```

Comment:
Nit: curlies around method body

Reply:
Done. I also rebased changes onto master to resolve conflicts.